docs/examples/cloud_trace_exporter/client.py

```diff
@@ -0,0 +1,34 @@
+# Copyright OpenTelemetry Authors
```

Comment:
is this example valuable? it seems like this is just outlining the user of the CloudTracerSpanExporter as the SpanExporter, which is illustrated in the basic_trace example above.

This is just illustrated that and the requests instrumentation. I worry that keeping this documentation will require updates for this package as requests instrumentation also changes API.